sparsity/dask/core.py

```diff
@@ -209,7 +217,7 @@ def join(self, other, on=None, how='left', lsuffix='',
     def to_npz(self, filename, blocksize=None,
                storage_options=None, compute=True):
         from sparsity.dask.io import to_npz
-        to_npz(self, filename, blocksize, storage_options, compute)
+        return to_npz(self, filename, blocksize, storage_options, compute)
```

Comment:
Could you please add `as something` to the import above or do something else to prevent naming confusion? I know it works now, but it would be prettier.